OxfordBodleian/Juel-Jensen/BDLaethg30.xml

```diff
@@ -107,39 +138,74 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                    part of the word <foreign xml:lang="gez">ሰላም፡</foreign>.
+                                    The elements of the punctuation signs were possibly rubricated (cp. on <locus target="#23v"/>).
+                                </seg>
                             </handNote>
                         </handDesc>
                         
                        <!-- <decoDesc>
-                         
                         </decoDesc>-->
                         
-                     <!--   <bindingDesc>
-                            <binding xml:id="binding">
-                                <decoNote xml:id="b1"></decoNote>
+                        <additions>
+                            <list>
+                                <item xml:id="e1">
+                                    <desc>A white sticker is glued to the inner side of the front board. The name of <persName ref="PRS5782JuelJen"/> is printed on it
+                                        (<foreign xml:lang="gez">ቤንት፡ ዩውል፡ የንሰን።</foreign>) and the signature "MS. Aeth. g. 30" is written in pencil below it. 
+                                        An additional sticker, with the signature "Ms 7" on it, is also glued to the inner side of the front board.
+                                        The textual content of the book ("Ethiopic Prayers") is indicated in pencil in the inner side of the front board. <!-- Can you decipher the second word? -->
+                                    </desc>
+                                </item>
+                                <item xml:id="e2">
+                                    <locus target="#2v"/>
+                                    <desc>Illegible writings in black ink, possibly in the main hand. The ink is almost completely faded out.</desc>
+                                </item>
+                            </list>
+                        </additions>
+                        <bindingDesc>
+                            <binding contemporary="true">
+                                <decoNote xml:id="b1">Two <material key="wood">wooden</material> boards.
+                                </decoNote>
                                 <decoNote xml:id="b2" type="bindingMaterial">
-                           <material key="wood"/>
-                        </decoNote>
+                                    <material key="wood"/>       
+                                </decoNote>
+                                <decoNote xml:id="b4" type="SewingStations">2</decoNote>
+                                <decoNote xml:id="b5" type="Spine">The spine cover is missing.</decoNote>
+                                <decoNote xml:id="b6" type="SlipCase">The manuscript is provided with a reddish-brown <material key="leather"/> slip case broken into two pieces.</decoNote>
```

Comment:
I think it is not broken but made of two (coarse) pieces. I have already seen other similar examples...I need to check it though.

Reply:
Good, I modified the description accordingly. For the binding part I consider it still preliminary, you might add more information when you revise it.